set.mm

```diff
@@ -27556,13 +27561,15 @@ generally appear in a single form (either definitional, but more often
   $}
 
   ${
-    $d x y $.  $d y A $.
-    nfcriv.1 $e |- F/_ x A $.
-    $( Consequence of the not-free predicate, similiar to ~ nfcri .  Requires
-       ` y ` and ` A ` be disjoint, but is not based on ~ ax-13 .  (Contributed
```

Comment:
Was this theorem not depending on ax-13 before, but is now? Then the old theorem should be kept!

Reply:
Neither spw, spvv nor nfcr depends on ax-13.